modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutorTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.sqltests;
+
+import java.util.Arrays;
+import java.util.List;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.internal.IgniteEx;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class GridMapQueryExecutorTest extends BaseSqlTest {
+    
+    /**
+     * Sets up data specific for these tests.
+     */
+    @Override
+    protected void setupData() {
+        super.setupData();
+        // Additional setup for GridMapQueryExecutor, if needed
```

Comment:
extra comment could be removed.
Possibly remove the method if no extra setup is required.

Reply:
Removed the redundant 'setupData' method as suggested. Thanks for the note.